nautobot/docs/installation/selinux-troubleshootiing.md

```diff
@@ -0,0 +1,199 @@
+Source RPM Packages           nginx-1.14.1-9.module_el8.0.0+184+e34fea82.x86_64
+Target RPM Packages
+SELinux Policy RPM            selinux-policy-targeted-3.14.3-54.el8_3.2.noarch
+Local Policy RPM              selinux-policy-targeted-3.14.3-54.el8_3.2.noarch
+Selinux Enabled               True
+Policy Type                   targeted
+Enforcing Mode                Enforcing
+Host Name                     nautobot.example.com
+Platform                      Linux nautobot.example.com
+                              4.18.0-240.1.1.el8_3.x86_64 #1 SMP Thu Nov 19
+                              17:20:08 UTC 2020 x86_64 x86_64
+Alert Count                   5
+First Seen                    2021-02-26 15:16:55 UTC
+Last Seen                     2021-02-26 15:23:12 UTC
+Local ID                      b83bb817-85f6-4f5c-b6e0-eee3acc85504
+
+Raw Audit Messages
+type=AVC msg=audit(1614352992.209:585): avc:  denied  { name_connect } for  pid=67245 comm="nginx" dest=8001 scontext=system_u:system_r:httpd_t:s0 tcontext=system_u:object_r:unreserved_port_t:s0 tclass=tcp_socket permissive=0
+
+
+type=SYSCALL msg=audit(1614352992.209:585): arch=x86_64 syscall=connect success=no exit=EACCES a0=12 a1=55d061477358 a2=10 a3=7ffc0c62296c items=0 ppid=67243 pid=67245 auid=4294967295 uid=988 gid=985 euid=988 suid=988 fsuid=988 egid=985 sgid=985 fsgid=985 tty=(none) ses=4294967295 comm=nginx exe=/usr/sbin/nginx subj=system_u:system_r:httpd_t:s0 key=(null)ARCH=x86_64 SYSCALL=connect AUID=unset UID=nginx GID=nginx EUID=nginx SUID=nginx FSUID=nginx EGID=nginx SGID=nginx FSGID=nginx
+
+Hash: nginx,httpd_t,unreserved_port_t,tcp_socket,name_connect
+```
+
+In the first few lines of the audit, `sealert` details what SELinux is blocking and provides some options to remedy the issue.
+Since Nginx is communicating with the Nautobot application via HTTP, the second option is the best option.
+
+```no-highlight
+SELinux is preventing /usr/sbin/nginx from name_connect access on the tcp_socket port 8001.
+
+*****  Plugin connect_ports (85.9 confidence) suggests   *********************
+
+If you want to allow /usr/sbin/nginx to connect to network port 8001
+Then you need to modify the port type.
+Do
+# semanage port -a -t PORT_TYPE -p tcp 8001
+    where PORT_TYPE is one of the following: dns_port_t, dnssec_port_t, kerberos_port_t, ocsp_port_t.
+
+*****  Plugin catchall_boolean (7.33 confidence) suggests   ******************
+
+If you want to allow httpd to can network connect
+Then you must tell SELinux about this by enabling the 'httpd_can_network_connect' boolean.
+
+Do
+setsebool -P httpd_can_network_connect 1
+```
+
+Executing `setsebool -P httpd_can_network_connect 1` should remedy the issue. Verify this by executing the `setsebool` command, verify that SELinux is enabled and in `enforcing` mode via the `sestatus` command, and test the application stack for functionality.
+
+```no-highlight
+# curl -ik https://nautobot.example.com
+HTTP/1.1 502 Bad Gateway
```

Comment:
Might be good to add a comment to the effect that this first attempt is showing the failure again - I did a bit of a double-take on seeing an error message here after the above paragraph explains the commands that should fix the issue.